llvm/include/llvm/SandboxIR/SandboxIR.h

```diff
@@ -256,6 +256,9 @@ class Value {
   template <typename ItTy, typename SBTy> friend class LLVMOpUserItToSBTy;
 
   Value(ClassID SubclassID, llvm::Value *Val, Context &Ctx);
+  /// Disable copies.
```

Comment:
When I tried to use `tryTrack()` in `PHINode::setIncomingValue()` I realized that `PHISetIncoming`'s constructor was taking a copy of the the instruction, rather than a reference, which would cause a crash when reverting. So I disabled copies here and I also reverted back to the original tracking code for `setIncomingValue()`. I will address those in a separate patch.